sound/soc/sof/sof-spi-dev.c

```diff
@@ -22,12 +22,14 @@
 #include "hw-spi.h"
 #include "ops.h"
 
+#if 0
```

Comment:
Why, can this part be just deleted?

Reply:
I think we can delete this, and add it back when PM on it is supported.

Reply:
done